Gruntfile.js

```diff
@@ -125,6 +125,10 @@ module.exports = function(grunt)
             'repository'
           ],
           overrides: {
+            main: ["js/kurento-utils.min.js"],
+            dependencies: {
+              "adapter.js": "*"
```

Comment:
```adapter.js``` is a temporal, external polyfill that will not be needed to use in the future, so it's up to the developer to add it or use their own one.